src/test/java/org/junitpioneer/internal/PioneerDateUtilsTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016-2021 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.internal;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("Pioneer date utilities")
+class PioneerDateUtilsTest {
```

Comment:
These tests are not discovered by our build. Please rename this class as `PioneerDateUtilsTests`.

Reply:
Done.